src/app/docs/button/button-docs.component.html

```diff
@@ -161,28 +184,37 @@
       <sky-docs-design-guideline
       >
         <p>
-          For buttons with only one possible direct object such as <strong>Add</strong> buttons in record list toolbars, use the following format for button text:
+          For buttons with only one possible direct object such as <strong *skyThemeIf="'default'">Add</strong><strong *skyThemeIf="'modern'">New</strong> buttons in record list toolbars, use the following format for button text:
         </p>
         <p>
           <strong>&lt;Verb&gt;</strong>
         </p>
-        <sky-docs-design-guideline-thumbnail
+        <sky-docs-design-guideline-thumbnail *skyThemeIf="'default'"
           imageSource="~/assets/img/guidelines/button/buttons-usage-1.png"
           >
         </sky-docs-design-guideline-thumbnail>
+        <sky-docs-design-guideline-thumbnail  *skyThemeIf="'modern'"
+          imageSource="~/assets/img/guidelines/button/buttons-usage-1-modern.png"
+          >
+        </sky-docs-design-guideline-thumbnail>
       </sky-docs-design-guideline>
 
       <sky-docs-design-guideline>
-        <p class="sky-docs-text-block">
-          For buttons with multiple possible direct objects such as <strong>Add alert</strong> buttons in page summary toolbars, use the following format for button text:
+        <p *skyThemeIf="'default'" class="sky-docs-text-block">
+          For buttons with multiple possible direct objects such as <strong >Add alert</strong> buttons in page summary toolbars, use the following format for button text:
```

Comment:
```suggestion
          For buttons with multiple possible direct objects, such as <strong >Add alert</strong> buttons in page summary toolbars, use the following format for button text:
```

Reply:
@Blackbaud-MattGregg, can we please merge this suggestion for consistency with the grammar edit in the preceding paragraph?